owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -466,5 +470,6 @@ class PassCodeActivity : AppCompatActivity() {
 
         const val EXTRAS_MIGRATION = "PASSCODE_MIGRATION"
         const val PASSCODE_MIN_LENGTH = 4
+        const val EXTRAS_PASSCODE_ENFORCED = "EXTRAS_PASSCODE_ENFORCED"
```

Comment:
If you apply the changes I suggested in the previous comments, maybe you can move this constant to a more general class, such as `SettingsSecurityFragment`. It may increase coupling a bit, but makes more sense as it is analogous for both classes in the same way. What do you think? 🤔

Reply:
You saved my day! ❤️   I was looking for a general class. But I didn't find it.